pkg/block/block_darwin.go

```diff
@@ -207,6 +207,11 @@ func storageControllerFromPlist(infoPlist *diskUtilInfoPlist) StorageController
 }
 
 func (info *Info) load() error {
+	if !info.ctx.EnableTools {
+		// we need to be verbose here
+		info.ctx.Warn("Skip EnableTools=false on darwin: would disable block support entirely.")
```

Comment:
This isn't actually skipping anything, though, right? I mean, we're not returning after this warning...

Reply:
yep, got lost in the rebase :\
my fault, will fix